upgrade-tidb-using-tiup.md

```diff
@@ -194,6 +194,27 @@ tiup cluster upgrade <cluster-name> v7.4.0
 >   3. reload 整个集群：`tiup cluster reload <cluster-name>`。此时，TiFlash 也会正常启动，无需额外操作。
 > - 在对使用 TiDB Binlog 的集群进行滚动升级过程中，请避免新创建聚簇索引表。
 
+#### 升级时指定组件版本
+
+从 tiup-cluster v1.14.0 开始，支持在升级集群的时候指定其中某些组件到特定版本。指定的这些组件会被固定到指定的版本，在下次升级中不会改变版本号，除非再次指定版本。
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup cluster upgrade -h | grep "version string"
+      --alertmanager-version string        Fix the version of alertmanager and no longer follows the cluster version.
+      --blackbox-exporter-version string   Fix the version of blackbox-exporter and no longer follows the cluster version.
+      --cdc-version string                 Fix the version of cdc and no longer follows the cluster version.
+      --ignore-version-check               Ignore checking if target version is bigger than current version
```

Comment:
```suggestion
      --ignore-version-check               Ignore checking if target version is bigger than current version.
```

Reply:
these lines are copied from promgram output,I think we should not change it until we fix it in program